.github/workflows/ci-actions.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           key: q2maven-${{ steps.get-date.outputs.date }}
       - name: Build
         run: |
-          mvn -e -B -DskipTests -DskipITs -Dno-format clean install
+          mvn -e -B -DskipTests -DskipITs -Dinvoker.skip -Dno-format -Dgib -Dgib.disable clean install
```

Comment:
Note:
- `-Dinvoker.skip` is offtopic but still worthwhile (skip maven invoker tests, e.g. in enforcer-rules)
- `-Dgib -Dgib.disable` just triggers the download of GIB and it's dependencies

Reply:
I wonder if we should replace `DskipTests -DskipITs -Dno-format` with `-Dquickly`

Reply:
IIRC, a few weeks ago someone said: `-Dquickly` is only for local usage.
But yeah, I also do see your point.

Reply:
-Dquickly skips the formatting check though. There seem to be tests that use maven invoker, won't they be affected by `-Dinvoker.skip`?

Reply:
> There seem to be tests that use maven invoker, won't they be affected by `-Dinvoker.skip`?

Yes, that's what should happen. This initial JDK11 build shall only _build_ everything and populate a local Maven repo that is then reused by subsequent jobs. This initial job shall not execute _any_ tests (see also `-DskipTests`), AFAICS.

Reply:
Please do not use `-Dquickly` in CI build.

This thing will evolve depending on what people need for local usage and we don't want CI to be impacted.

Reply:
Btw, the docs job _is_ using `-Dquickly`: https://github.com/quarkusio/quarkus/blob/master/.github/workflows/doc-build.yml#L45